tests/acceptance-bdd/test_literature_suggestion_form.py

```diff
@@ -0,0 +1,392 @@
+        EC.visibility_of_element_located((By.XPATH, '//div[@id="webdeposit_form_accordion"]/div[4]/span/button'))
+    ).click()
+
+
+@when('I click on the accept button')
+def click_accept_button():
+    WebDriverWait(Arsenic(), 2).until(
+        EC.visibility_of_element_located((By.XPATH, '//button[@class="btn btn-warning"]'))
+    ).click()
+
+
+@when('I click on the button Skip, and fill the form manually')
+def click_skip_and_fill_manually():
+    Arsenic().hide_title_bar()
+    WebDriverWait(Arsenic(), 5).until(
+        TryClick((By.ID, 'skipImportData'))
+    ).click()
+    WebDriverWait(Arsenic(), 5).until(
+        EC.text_to_be_present_in_element(
+            (By.ID, 'form_container'),
+            'Type of Document',
+        )
+    )
+    Arsenic().find_element_by_link_text('Conference Information').click()
+    Arsenic().execute_script(
+        """document.evaluate(
+            "//div[@id='webdeposit_form_accordion']/div[3]/div[7]/div[1]",
+            document,
+            null,
+            XPathResult.FIRST_ORDERED_NODE_TYPE,
+            null
+        ).singleNodeValue.click()"""
+    )
+    Arsenic().execute_script(
+        """document.evaluate(
+            "//div[@id='webdeposit_form_accordion']/div[3]/div[8]/div[1]",
+            document,
+            null,
+            XPathResult.FIRST_ORDERED_NODE_TYPE,
+            null
+        ).singleNodeValue.click()"""
+    )
+    Arsenic().execute_script(
+        """document.evaluate(
+            "//div[@id='webdeposit_form_accordion']/div[3]/div[9]/div[1]",
+            document,
+            null,
+            XPathResult.FIRST_ORDERED_NODE_TYPE,
+            null
+        ).singleNodeValue.click()"""
+    )
+    Arsenic().show_title_bar()
+
+
+@when(parsers.cfparse('I select the value {input_select} in the select box with id {field_id}'))
+def select_generic_value(input_select, field_id):
+    Select(Arsenic().find_element_by_id(field_id)).select_by_value(input_select)
+
+
+@when(parsers.cfparse('I select the value {input_select} in the subject dropdown list'))
+def select_generic_value_dropdown(input_select):
+    Arsenic().find_element_by_xpath('(//button[@type="button"])[8]').click()
+    Arsenic().find_element_by_css_selector('input[type=\"checkbox\"]').click()
+    Arsenic().find_element_by_xpath(
+        '//input[@value="' + input_select + '"]'
+    ).click()
+    Arsenic().find_element_by_xpath('(//button[@type="button"])[8]').click()
+
+
+@then(parsers.cfparse('I should see the {field} error message'))
+def is_there_a_message_error(field):
+    assert globals()['_is_' + field + '_error_message_present']()
+
+
+@then(parsers.cfparse('I should not see the {field} error message'))
+def is_not_there_a_message_error(field):
+    assert not globals()['_is_' + field + '_error_message_present']()
+
+
+@when(parsers.cfparse('I insert <input> in the input box with id {field_id}'))
+@when(parsers.cfparse('I insert {input} in the input box with id {field_id}'))
```

Comment:
We need both because with that function we are able to handle 2 different statement syntax.